src/test/java/com/hydratereminder/HydrateReminderImagesTest.java

```diff
@@ -42,11 +39,18 @@
      * <p>Tests that the correct ID is generated for an image
      * </p>
```

Comment:
Your changes look good @ddanishkhan!

Can you just update the comment of the test method to include details on the params `expectedItemID` and `image`? Afterwards, I'll approve it and get it merged in.

Reply:
@jmakhack Added the JavaDoc comments